src/config/dynamic-theme-fixes.config

```diff
@@ -5097,6 +5097,20 @@ INVERT
 
 ================================
 
+theinformation.com
+
+INVERT
+.logo
```

Comment:
Seems to me it's already inverted without this fix 
![](https://i.imgur.com/ial1Ssn.png)

Reply:
Good catch. I only looked at the article pages on the other side of the paywall. It looks like the gated homepage is the only page with a logo that inverts correctly. Adding some CSS will fix the box-shadows on the gated homepage too. Pasting my new devtools code below.

```suggestion
theinformation.com

INVERT
.logo

CSS
article.single .sticky-header button.button-share,
#bookmark-and-share button.button-share {
    background-color: #21498c;
    color: #e8e6e3;
}
article.single .sticky-header button.button-share:hover,
#bookmark-and-share button.button-share:hover {
    background-color: #245098;
}
.locked section .insert-container {
    -webkit-filter: drop-shadow(20px 20px 20px #26292b);
    filter: drop-shadow(20px 20px 20px #26292b);
}
.locked section .insert-container#app-screenshot-container {
    -webkit-filter: drop-shadow(20px -20px 20px #26292b);
    filter: drop-shadow(20px -20px 20px #26292b);
}

IGNORE IMAGE ANALYSIS
header.locked .logo

================================

```

Screenshot of article page with this code applied.

<img width="1089" alt="Screen Shot 2020-08-17 at 11 46 03 AM" src="https://user-images.githubusercontent.com/25544113/90415775-93997e80-e07f-11ea-8be9-2985b0225764.png">

Reply:
You can drop the `-webkit-filter` It's not needed for fixes for the rest it LGTM make sure check for `light` scheme in dark reader if it don't break the site.

Reply:
Thanks. Double-checked the light theme and committed changes with `-webkit-filter` dropped.